packages/chargebee/src/portal_link/index.tsx

```diff
@@ -1,19 +1,34 @@
 import PropTypes from 'prop-types';
 
 import * as sections from '../constants';
-import Action from '../action';
+import {
+  ChargebeeForwardOptions,
+  PortalSectionsKeys,
+  GetPortalSessionFunction,
+  OnCallHandler,
+  VoidFunction,
+} from '../types';
+import Action, { ActionProps } from '../action';
+
+type PortalLinkProps = ActionProps & {
```

Comment:
```suggestion
type PortalLinkProps = Omit<ActionProps, 'section' | 'onSessionGet' | 'onClose'> & {
```

Reply:
Why omit when `ActionProps` doesn't include any of those values?

Reply:
ok seems like now you only need to omit `onCall`